src/test/resources/epub3/mediaoverlays-publication.feature

```diff
@@ -22,7 +22,14 @@ Feature: EPUB 3 ▸ Media Overlays ▸ Full Publication Checks
 
   ## 2. Media Overlay Document Definition
 
-  ### The audio Element
+  Scenario: Report invalid fragment identifiers on body, seq, and text elements
+    When checking EPUB 'mediaoverlays-invalid-fragid-error'
+    Then error MED-014 is reported
+    And error MED-015 is reported
+    And error RSC-012 is reported
+    And no other errors or warnings are reported
```

Comment:
I suggest the test is split in two:
- one test for missing or empty fragment id (`mediaoverlays-text-src-fragment-missing-error`), to test `MED-014/MED-015`
- one test for a fragment not found in the content doc (`mediaoverlays-text-src-fragment-not-found-in-content-error`), to test `RSC-012`